Sources/SBTUITestTunnelServer/SBTUITestTunnelServer.m

```diff
@@ -1263,6 +1263,8 @@ - (NSDictionary *)commandCoreLocationStubServiceStatus:(NSDictionary *)parameter
 
 - (NSDictionary *)commandCoreLocationNotifyUpdate:(NSDictionary *)parameters
 {
+    [self commandCoreLocationStubManagerLocation:parameters];
```

Comment:
Could make sense to avoid this, in order to allow `locationNotifyUpdate` without side effects on `location`?

Reply:
I believe this is the more expected behavior. Doing otherwise would require from a test perspective to always update the cllocationmanger location as well with doesn't feel particularly ergonomic.